tests/phpunit/tests/fonts/font-face/wpFontFaceResolver/getFontsFromThemeJson.php

```diff
@@ -37,20 +37,58 @@ public function test_should_return_all_fonts_from_theme() {
 		$this->assertSame( $expected, $actual );
 	}
 
+	/**
+	 * @ticket 60605
+	 */
+	public function test_should_return_all_fonts_from_all_theme_origins() {
+		switch_theme( static::FONTS_THEME );
+
+		$add_custom_fonts = function ( $theme_json_data ) {
+			$data = $theme_json_data->get_data();
+			// Add font families to the custom origin of theme json.
+			$data['settings']['typography']['fontFamilies']['custom'] = $this->get_custom_font_families( 'input' );
```

Comment:
Whoopsie, a function within a method does not understand `$this`. 

How to fix? Move this function callback to be a method instead.

Sorry, I didn't notice it when I suggested moving to a variable.

Reply:
This commit fixes it https://github.com/WordPress/wordpress-develop/commit/fea390858e2e0e58fb170795360c868e2d165844